communities/models.py

```diff
@@ -47,8 +47,8 @@ class PlatformEnum(models.TextChoices):
     )
     link = models.URLField(_("Link"), unique=True)
     date = models.DateField(_("Date"), auto_now_add=True)
-    category = TextChoicesField(choices_enum=CategoryEnum, default="")
-    platform = TextChoicesField(choices_enum=PlatformEnum, default="")
+    category = TextChoicesField(choices_enum=CategoryEnum, default="")  # type: ignore
```

Comment:
not sure why the `# type: ignore` is doing

Reply:
mypy is flagging an error because of the empty string, that's how I disabled it

Reply:
aha, im also using `Django-pylint` an extension for `pylint` not sure if will resolve that